.python-version

```diff
@@ -0,0 +1 @@
+3.10
```

Comment:
It turns out that if we use 3.11 or newer than pandas needs to be built from sources. I decided to stick with 3.10 for now (= faster builds) and deal with the dependency updates at some point in the future (pandas, numpy, plotly... it's all connected apparently).

Reply:
Found an easy way to reproduce this issue:

3.11

```
$ uvx --no-build --with="pandas==1.3.5" --python=3.11 pip list
  × No solution found when resolving tool dependencies:
  ╰─▶ Because pandas==1.3.5 has no usable wheels and you require pandas==1.3.5, we can conclude that your requirements are unsatisfiable.
  ```

3.10
  
```
  $ uvx --no-build --with="pandas==1.3.5" --python=3.10 pip list
Package         Version
--------------- -----------
numpy           2.2.6
pandas          1.3.5
pip             25.2
python-dateutil 2.9.0.post0
pytz            2025.2
six             1.17.0
```

Reply:
The problem can be solved by using pandas 2.x and the dependants seems to be ok with it but I'll postpone that change for now.